src/lib/roles.ts

```diff
@@ -326,16 +326,22 @@ export const roles_definition = validateRoles({
 		"write":false
 	},
 	"value.blind":{
-		"desc":"Blind position",
+		"desc":"Actual position of blind (100% = fully open, 0% = fully closed)",
 		"type":"number",
 		"read":true,
-		"write":false
+		"write":false,
+		"unit": "%",
+		"min": 0,
+		"max": 100
 	},
 	"value.tilt":{
-		"desc":"Tilt position",
+		"desc":"Actual tilt position (100% = fully open, 0% = fully closed)",
```

Comment:
I would remove "position" from the tilt description. Blinds have a position (up down) and tilt.